.github/security-insights.yml

```diff
@@ -0,0 +1,25 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-04-02'
+  last-reviewed: '2025-04-02'
+  project-si-source: https://github.com/open-telemetry/.github/security-insights.yml
```

Comment:
I got a 404 HTTP status code from  `https://github.com/open-telemetry/.github/security-insights.yml`. Is this expected?

Reply:
The URL should be https://github.com/open-telemetry/.github/blob/main/security-insights.yml